### DISCUSSION TITLE
Coordinator_proxy problem and weak system

### DISCUSSION THREAD

**Discussion by BrainKiller-Crypto** [subquery/subql#1053] [Frontier TestNet - Bug Reporting]
**Coordinator_proxy problem**
**After pulling the images from Docker and starting the coordinator_proxy service, the connection to coordinator_service and websocket is successful.**
![photo_2022-05-22 01 32 49](https://user-images.githubusercontent.com/79287195/169671033-1db17e90-9ab6-4c35-a90e-55e350177c9c.jpeg)

**After that, errors appear in the coordinator_proxy logs.**

![photo_2022-05-22 01 34 38](https://user-images.githubusercontent.com/79287195/169671072-b212ac1b-b6c3-49e6-9809-a7207883b16d.jpeg)

                                                                                      **weak system**
![2022-05-22 в 0 49 18](https://user-images.githubusercontent.com/79287195/169671365-bda3c665-3f06-4eea-924a-cc3f80ae8a5b.png)
![photo_2022-05-22 01 42 59](https://user-images.githubusercontent.com/79287195/169671367-b39901ef-daaf-465e-b881-5f1c3dfcf460.jpeg)

**Comment by mzxyz** [comment]
Checkout this: https://discord.com/channels/796198414798028831/958834228898967603/973720873141927966 https://discord.com/channels/796198414798028831/958834228898967603/972361940833812520 Hope these are helpful
